build.sbt

```diff
@@ -18,7 +19,6 @@ scalacOptions in ThisBuild := Seq(
   "-unchecked",
   "-Ywarn-dead-code",
   "-Ywarn-numeric-widen",
-  "-Ywarn-unused-import",
```

Comment:
why? :(

Reply:
Got an unsupported flag after switching to scala 2.13